<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>智能餐饮管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="toptiplabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>2</y>
      <width>1024</width>
      <height>25</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>欢迎使用本系统</string>
    </property>
   </widget>
   <widget class="QTableView" name="tableView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>64</y>
      <width>771</width>
      <height>708</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QFrame" name="funframe">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>28</y>
      <width>770</width>
      <height>34</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>2</y>
       <width>225</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>8</y>
       <width>54</width>
       <height>19</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>快捷码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="refreshbtn">
     <property name="geometry">
      <rect>
       <x>636</x>
       <y>0</y>
       <width>75</width>
       <height>34</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>刷新同步</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>712</x>
       <y>0</y>
       <width>57</width>
       <height>34</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">border-color: rgb(255, 255, 255);</string>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToMinimumContentsLength</enum>
     </property>
     <property name="duplicatesEnabled">
      <bool>true</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>缩图</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>列表</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>简字</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>472</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>test</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView_2">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>176</y>
      <width>252</width>
      <height>544</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="settlebtn">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>720</y>
      <width>252</width>
      <height>49</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>结算打印</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createbtn">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>28</y>
      <width>252</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>新增流水单</string>
    </property>
   </widget>
   <widget class="QPushButton" name="querybtn">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>68</y>
      <width>252</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>最近流水单查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settingbtn">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>108</y>
      <width>252</width>
      <height>39</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>功能与设置</string>
    </property>
   </widget>
   <widget class="QFrame" name="orderIDlabel">
    <property name="geometry">
     <rect>
      <x>772</x>
      <y>148</y>
      <width>252</width>
      <height>28</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>78</width>
       <height>26</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="text">
      <string>订单流水号</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="margin">
      <number>1</number>
     </property>
     <property name="buddy">
      <cstring></cstring>
     </property>
    </widget>
    <widget class="QComboBox" name="orderIDCombobox">
     <property name="geometry">
      <rect>
       <x>76</x>
       <y>-2</y>
       <width>174</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
       <weight>9</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;黑体&quot;;</string>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="maxVisibleItems">
      <number>20</number>
     </property>
     <property name="duplicatesEnabled">
      <bool>true</bool>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
